android/assets/jsons/translations/Polish.properties

```diff
@@ -2421,7 +2421,7 @@ Discover [tech] = Odkrywa [tech]
 Adopt [policy] = Przyjmuje [policy]
 Remove [policy] = Zrezygnuj z ustroju [policy]
  # Requires translation!
-Remove [policy] and refund [amount]% of its cost = 
+Remove [policy] and refund [amount]% of its cost = Zrezygnuj z ustorju społecznego [policy] i otrzymaj z powrotem [amount]% jego kosztu
```

Comment:
I noticed a typo, "ustorju" should be changed to "ustroju".

Reply:
Fixed. Thanks for reviewing!